style.css

```diff
@@ -1,340 +1,656 @@
+.catalog__wrapper {
+    display: flex;
+}
+
+.catalog__actions .btn--margin-right {
+    position: relative;
+    margin-right: 32px;
+}
+
+.catalog__actions .btn--margin-right:after {
+    content: '';
+    position: absolute;
+    top: 10px;
+    right: -18px;
+    width: 2px;
+    height: 24px;
+    background-color: #D0D7FB2B;
+}
+
+.catalog__title {
+    position: relative;
+    padding-left: 30px;
+    font-family: Bakbak One;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+}
+
+.catalog__title::before {
+    position: absolute;
+    content: '';
+    top: 0px;
+    left: 0px;
+    width: 24px;
+    height: 24px;
+    background-image: url('images/folder.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+.catalog__title::after {
+    position: absolute;
+    content: '';
+    top: 15px;
+    right: 0px;
+    width: 90%;
+    height: 2px;
+    border-radius: 2px;
+    background-color: #D0D7FB2B;
+}
+
+.catalog__item {
+    display: flex;
+    flex-direction: column;
+    padding: 30px;
+    margin-right: 24px;
+    background-color: #1D1C28;
+}
+
+.catalog__image {
+    position: relative;
+}
+
+.catalog__price {
+    font-family: Bakbak One;
+    font-size: 36px;
+    font-weight: 400;
+    line-height: 48px;
+    letter-spacing: -0.5px;
+}
+
+.catalog__name {
+    font-family: Poppins;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+    color: #EFF5FA;
+}
+
+/* Index page Blog block */
+
+.blog__wrapper {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
 }
+
+.blog__post {
+    display: grid;
+    grid-row: span 2;
+    height: 900px;
+    padding: 30px;
+    margin-right: 30px;
+    margin-bottom: 30px;
+    background-color: #1D1C28;
+}
+
+.blog__image {
+    position: relative;
+    height: 640px;
+}
+
+.blog__post--small {
+    height: 435px;
+}
+
+.blog__post--small .blog__image {
+    height: 175px;
+    overflow: hidden;
+}
+
+.blog__info {
+    display: flex;
+}
+
+.blog__name,
+.blog__date {
+    position: relative;
+    padding-left: 30px;
+}
+
+.blog__name::before {
+    position: absolute;
+    content: '';
+    background-image: url('images/user.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    top: 0px;
+    left: 0px;
+    width: 16px;
+    height: 16px;
+}
+
+.blog__date::before {
+    position: absolute;
+    content: '';
+    background-image: url('images/calendar.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    top: 0px;
+    left: 0px;
+    width: 16px;
+    height: 16px;;
+}
+
+/* Index page Our Team block */
+
+.team__members {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    padding: 0 150px;
+}
+
+.team__item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 316px;
+    height: 293px;
+    margin: 0 30px 30px 0;
+    padding: 30px;
+    background-color: #1D1C28;
+}
+
+.team__image {
+    display: block;
```

Comment:
Just FYI: img have  indent (or 'jumps'), because img inline-block by default and we need set vertical-align: top; 
But you style display: block; in this case also can resolve this problem. 
As usual, when you try normalize css you set globally (in the top of your css file) `img { vertical-align: top; }`

Don't change something here.

Reply:
okey, understand!